src/main/java/org/dependencytrack/parser/snyk/SnykParser.java

```diff
@@ -204,45 +208,61 @@ public List<VulnerableSoftware> parseVersionRanges(final QueryManager qm, final
             for (String part : parts) {
                 if (part.startsWith(">=") || part.startsWith("[")) {
                     versionStartIncluding = part.replace(">=", "").replace("[", "").trim();
+                    if (versionStartIncluding.length() == 0 || versionStartIncluding.contains("*")) {
+                        versionStartIncluding = null;
+                    }
                 } else if (part.startsWith(">") || part.startsWith("(")) {
                     versionStartExcluding = part.replace(">", "").replace("(", "").trim();
+                    if (versionStartExcluding.length() == 0 || versionStartExcluding.contains("*")) {
+                        versionStartIncluding = null;
+                    }
                 } else if (part.startsWith("<=") || part.endsWith("]")) {
                     versionEndIncluding = part.replace("<=", "").replace("]", "").trim();
                 } else if (part.startsWith("<") || part.endsWith(")")) {
                     versionEndExcluding = part.replace("<", "").replace(")", "").trim();
+                    if (versionEndExcluding.length() == 0 || versionEndExcluding.contains("*")) {
+                        versionStartIncluding = null;
+                    }
                 } else if (part.startsWith("=")) {
                     versionStartIncluding = part.replace("=", "").trim();
                     versionEndIncluding = part.replace("=", "").trim();
-                } else {
-                    LOGGER.warn("Unable to determine version range " + part);
+                    if (versionStartIncluding.length() == 0 || versionStartIncluding.contains("*")) {
+                        versionStartIncluding = null;
+                    }
+                    if (versionEndIncluding.length() == 0 || versionEndIncluding.contains("*")) {
+                        versionStartIncluding = null;
+                    }
+                } else { //since we are not able to parse specific range, we do not want to end up with false positives and therefore this part will be skipped from being saved to db.
+                    LOGGER.warn("Range not definite. Not saving this vulnerable software information. The purl was: "+purl);
```

Comment:
Please lower this to `debug`, as there is nothing users can do against it, and it produces a lot of noise.

Reply:
done